spec/services/action_create_service_spec.rb

```diff
@@ -71,6 +77,13 @@
     end
   end
 
+  context 'auto set processed_by if nil' do
+    it 'create a new action with nil processed_by' do
+      action = svc1.create('operation' => Action::MEMO_OPERATION, 'comments' => 'later')
+      expect(action.processed_by).to eq("jdoe")
```

Comment:
should not hardcode `jdoe`, should be `ManageIQ::API::Common::Request.current.user.username`